app/src/main/java/nextstep/jwp/http/ContentTypeMapper.java

```diff
@@ -0,0 +1,26 @@
+package nextstep.jwp.http;
+
+import java.util.Arrays;
+
+public enum ContentTypeMapper {
+    HTML(".html", "text/html;charset=utf-8"),
+    CSS(".css", "text/css"),
+    JAVASCRIPT(".js", "application/javascript; charset=UTF-8"),
+    ICON(".ico", "image/x-icon");
+
+    String extension;
+    String contentType;
+
+    ContentTypeMapper(String extension, String contentType) {
+        this.extension = extension;
+        this.contentType = contentType;
+    }
+
+    public static String extractContentType(String path) {
+        return Arrays.stream(values())
+                .filter(contentType -> path.contains(contentType.extension))
```

Comment:
contains 보다는 `endsWith(...)`가 더 적절하지 않을까 싶어요!  (물론 지금 방식도 크게 문제없이 동작하겠지만요 👀)
```suggestion
                .filter(contentType -> path.endsWith(contentType.extension))
```

Reply:
👍